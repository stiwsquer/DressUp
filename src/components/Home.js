import React from "react";
import gallery1 from "../images/gallery-1.2.jpg";
import gallery2 from "../images/gallery-2.1.jpg";
import gallery3 from "../images/gallery-3.1.jpg";
import home_intro from "../images/home-intro_1.jpg";

export default function Home(props) {
  return (
    <>
      <section className="home-intro"></section>
      <section className="home-description">
        <h2>
          Trendy Clothing & Accessories at Dress Up - An Online Dress Boutique
        </h2>
        <p>
          DRESS UP IS A WOMEN&apos;S CLOTHING BOUTIQUE CARRYING A VARIETY OF
          FITS, STYLES, AND PATTERNS FOR WOMEN OF ALL AGES, SHAPES, AND SIZES.
          WE OFFER A DIVERSE COLLECTION OF NEW PRODUCTS EVERY SINGLE WEEK,
          INCLUDING YOUR FAVORITE CLOTHING ITEMS: CUTE ROMPERS, MAXI DRESSES,
          KNIT SWEATERS, OVERSIZED TOPS, AND LIGHTWEIGHT CARDIGANS, AS WELL AS
          TRENDY SHOES, ACCESSORIES, JEWELRY, ACTIVEWEAR, LOUNGEWEAR, AND SO
          MUCH MORE. DRESS UP&apos;S TRENDY, YET WEARABLE CLOTHES WILL HAVE YOU
          FEELING LIKE YOUR BEST SELF, NO MATTER WHERE YOU&apos;RE HEADED. YOU
          CAN SHOP WITH US ONLINE OR AT ANY OF OUR RETAIL LOCATIONS ACROSS THE
          SOUTHEAST, INCLUDING STORES IN GEORGIA, ALABAMA, SOUTH CAROLINA, AND
          TENNESSEE. GET FREE SHIPPING ON ALL ORDERS OVER $49, AND FEEL FREE TO
          REACH OUT TO US AT 678.989.2006 WITH ANY QUESTIONS. DON&apos;T MISS
          OUT - COME SEE WHY DRESS UP IS ONE OF THE FASTEST GROWING ONLINE DRESS
          BOUTIQUES IN THE UNITED STATES!{" "}
        </p>
        <button>SHOP OUR NEWEST ARRIVALS</button>
      </section>
      <section className="home-gallery">
        <div className="gallery-new-shoes">
          <h2>NEW SHOES</h2>
          <p>
            We just dropped new fall shoe styles, and they are all ON SALE! Shop
            new combat boots, booties, mules, sneakers & more!
          </p>
          <button>SHOP SHOES</button>
        </div>
        <div className="gallery-accessories">
          <h2>Accessories</h2>
          <p>
            Take your fall outfit to the next level with new accessories! The
            perfect trendy handbag, dainty gold necklace, coin bracelet or
            statement earrings will upgrade any look.
          </p>
          <button>SHOP ACCESSORIES</button>
        </div>
        <img className="gallery1" src={gallery1} />
        <img className="gallery2" src={gallery2} />
        <img className="gallery3" src={gallery3} />
      </section>
    </>
  );
}
