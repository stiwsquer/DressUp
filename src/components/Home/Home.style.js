import styled from "styled-components";
import homeIntroBackground from "../../static/images/home-intro_1.jpg";

export const HomeIntro = styled.section`
  min-height: 100vh;
  background-image: url("${homeIntroBackground}");
  background-repeat: no-repeat;
  background-size: cover;
  background-position: center;
`;
