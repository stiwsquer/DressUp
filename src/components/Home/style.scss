@import "../../styles/global.scss";
.home-intro {
  min-height: 100vh;
  background-image: url("../../images/home-intro_1.jpg");
  background-repeat: no-repeat;
  background-size: cover;
  background-position: center;
}
