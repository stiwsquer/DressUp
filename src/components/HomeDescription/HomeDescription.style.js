import styled from "styled-components";
import { flex } from "../../GlobalStyles";

export const HomeDescriptionSection = styled.section`
  min-height: 40vh;
  width: 95%;
  margin: auto;
  text-align: center;
  line-height: 3rem;

  ${flex({ direction: "column", justify: "center", align: "center" })}

  h2 {
    font-size: 2.5rem;
    margin: 4rem;
  }
  p {
    font-size: 1.5rem;
  }
  button {
    margin: 4rem;
  }
`;
