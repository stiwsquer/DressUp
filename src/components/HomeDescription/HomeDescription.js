import React from "react";
import Button from "../Button/Button";
import { HomeDescriptionSection } from "./HomeDescription.style";
export default React.memo(function HomeDescription() {
  return (
    <HomeDescriptionSection>
      <h2>
        Trendy Clothing & Accessories at Dress Up - An Online Dress Boutique
      </h2>
      <p>
        DRESS UP IS A WOMEN&apos;S CLOTHING BOUTIQUE CARRYING A VARIETY OF FITS,
        STYLES, AND PATTERNS FOR WOMEN OF ALL AGES, SHAPES, AND SIZES. WE OFFER
        A DIVERSE COLLECTION OF NEW PRODUCTS EVERY SINGLE WEEK, INCLUDING YOUR
        FAVORITE CLOTHING ITEMS: CUTE ROMPERS, MAXI DRESSES, KNIT SWEATERS,
        OVERSIZED TOPS, AND LIGHTWEIGHT CARDIGANS, AS WELL AS TRENDY SHOES,
        ACCESSORIES, JEWELRY, ACTIVEWEAR, LOUNGEWEAR, AND SO MUCH MORE. DRESS
        UP&apos;S TRENDY, YET WEARABLE CLOTHES WILL HAVE YOU FEELING LIKE YOUR
        BEST SELF, NO MATTER WHERE YOU&apos;RE HEADED. YOU CAN SHOP WITH US
        ONLINE OR AT ANY OF OUR RETAIL LOCATIONS ACROSS THE SOUTHEAST, INCLUDING
        STORES IN GEORGIA, ALABAMA, SOUTH CAROLINA, AND TENNESSEE. GET FREE
        SHIPPING ON ALL ORDERS OVER $49, AND FEEL FREE TO REACH OUT TO US AT
        678.989.2006 WITH ANY QUESTIONS. DON&apos;T MISS OUT - COME SEE WHY
        DRESS UP IS ONE OF THE FASTEST GROWING ONLINE DRESS BOUTIQUES IN THE
        UNITED STATES!
      </p>
      <Button>SHOP OUR NEWEST ARRIVALS</Button>
    </HomeDescriptionSection>
  );
});
