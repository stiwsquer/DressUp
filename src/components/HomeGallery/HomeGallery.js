import React from "react";
import Button from "../Button/Button";
import "./style.scss";

export default React.memo(function HomeGallery() {
  return (
    <section className="home-gallery">
      <div className="gallery-new-shoes">
        <h2>NEW SHOES</h2>
        <p>
          We just dropped new fall shoe styles, and they are all ON SALE! Shop
          new combat boots, booties, mules, sneakers & more!
        </p>
        <Button>SHOP SHOES</Button>
      </div>
      <div className="gallery-accessories">
        <h2>Accessories</h2>
        <p>
          Take your fall outfit to the next level with new accessories! The
          perfect trendy handbag, dainty gold necklace, coin bracelet or
          statement earrings will upgrade any look.
        </p>
        <Button>SHOP ACCESSORIES</Button>
      </div>
      <img className="gallery1" src="./img/home/gallery/gallery-1.2.jpg" />
      <img className="gallery2" src="./img/home/gallery/gallery-2.1.jpg" />
      <img className="gallery3" src="./img/home/gallery/gallery-3.1.jpg" />
    </section>
  );
});
