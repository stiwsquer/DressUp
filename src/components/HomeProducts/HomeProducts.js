import React, { useEffect, useRef, useState } from "react";
import Carousel from "react-elastic-carousel";
import "./style.scss";
import Card from "../Card/Card";
export default function HomeProducts() {
  const [cards, setCards] = useState([]);
  const breakPoints = [
    { width: 1, itemsToShow: 1 },
    { width: 500, itemsToShow: 2 },
    { width: 1000, itemsToShow: 3 },
    { width: 1300, itemsToShow: 4 },
    { width: 2100, itemsToShow: 6 },
    { width: 3000, itemsToShow: 7 },
    { width: 3500, itemsToShow: 8 },
  ];

  const getProducts = async () => {
    try {
      let products = await fetch("/products.json");
      let json = await products.json();
      const Cards = json.map((card, index) => {
        return (
          <Card
            key={index}
            linkTo="#"
            title={card[0].title}
            text={card[0].text}
            imagesSources={card}
          ></Card>
        );
      });
      setCards(Cards);
      return Cards;
    } catch (err) {
      console.log(err);
    }
  };
  useEffect(getProducts, []);

  // const imgs = [
  //   {
  //     src: "./img/products/1.JPG",
  //     srcHover: "./img/products/1.2.webp",
  //     imgAlt: "Image of women in a dress",
  //     backgroundColor: `url("https://cdn11.bigcommerce.com/s-3y69bgeked/images/stencil/768w/attribute_value_images/73764.preview.jpg?t=1630492842`,
  //   },
  //   {
  //     src: "./img/products/2.webp",
  //     srcHover: "./img/products/2.1.webp",
  //     imgAlt: "Image of women in a dress",
  //     backgroundColor: `url("https://cdn11.bigcommerce.com/s-3y69bgeked/images/stencil/768w/attribute_value_images/73764.preview.jpg?t=1630492842`,
  //   },
  // ];
  return (
    <section className="home-products">
      <div className="products-title">
        <span className="horizontal-line"></span>
        <h2>NEW PRODUCTS</h2>
        <span className="horizontal-line"></span>
      </div>
      {/**
       * @TODO move all products info to an external file and process it here
       */}
      <div className="cards">
        <Carousel breakPoints={breakPoints}>
          {cards}
          {/* <Card
            linkTo="#"
            title="Fiercely Fabulous Leopard Ruched Mini Dress"
            text="$44.00"
            imagesSources={imgs}
            // backgroundColor2='url("https://cdn11.bigcommerce.com/s-3y69bgeked/images/stencil/768w/attribute_value_images/73764.preview.jpg?t=1630492842")'
          ></Card> */}

          {/* <Card
            linkTo="#"
            imgSrcNormal={"./img/products/2.webp"}
            imgSrcAlt={"./img/products/2.1.webp"}
            imgAltNormal="Image of women in a dress"
            imgAltAlt="Image of women in a dress"
            title="Tame My Heart Leopard Tie-Front Dress"
            text="$40.00"
          ></Card>
          <Card
            linkTo="#"
            imgSrcNormal={"./img/products/3.webp"}
            imgSrcAlt={"./img/products/3.1.webp"}
            imgAltNormal="Image of women in a dress"
            imgAltAlt="Image of women in a dress"
            title="Keepin' It Chic Tiered Dress "
            text="$33.00"
          ></Card>
          <Card
            linkTo="#"
            imgSrcNormal={"./img/products/4.webp"}
            imgSrcAlt={"./img/products/4.1.JPG"}
            imgAltNormal="Image of women in a dress"
            imgAltAlt="Image of women in a dress"
            title="Walk The Walk Mini Skirt"
            text="$39.00"
          ></Card>
          <Card
            linkTo="#"
            imgSrcNormal={"./img/products/5.JPG"}
            imgSrcAlt={"./img/products/5.1.JPG"}
            imgAltNormal="Image of women in a dress"
            imgAltAlt="Image of women in a dress"
            title="Just The One Floral Dress - Mustard"
            text="$44.00"
          ></Card>
          <Card
            linkTo="#"
            imgSrcNormal={"./img/products/6.webp"}
            imgSrcAlt={"./img/products/6.1.webp"}
            imgSrcNormal2={"./img/products/6.2.webp"}
            imgSrcAlt2={"./img/products/6.3.JPG"}
            imgAltNormal="Image of women in a dress"
            imgAltAlt="Image of women in a dress"
            title="Iconic Style Tie-Front Blouse"
            text="$35.00"
            backgroundColor="blue"
            backgroundColor2='url("https://cdn11.bigcommerce.com/s-3y69bgeked/images/stencil/768w/attribute_value_images/73764.preview.jpg?t=1630492842")'
          ></Card>
          <Card
            linkTo="#"
            imgSrcNormal={"./img/products/7.JPG"}
            imgSrcAlt={"./img/products/7.1.JPG"}
            imgAltNormal="Image of women in a dress"
            imgAltAlt="Image of women in a dress"
            title="Wild Spirit Leopard Babydoll Dress"
            text="$41.00"
          ></Card>
          <Card
            linkTo="#"
            imgSrcNormal={"./img/products/8.webp"}
            imgSrcAlt={"./img/products/8.1.webp"}
            imgAltNormal="Image of women in a dress"
            imgAltAlt="Image of women in a dress"
            title="Wild Impression Leopard Blouse"
            text="$33.00"
          ></Card> */}
        </Carousel>
      </div>
    </section>
  );
}
