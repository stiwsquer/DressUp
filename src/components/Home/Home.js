import React from "react";
import "./style.scss";

import HomeDescription from "../HomeDescription/HomeDescription";
import HomeGallery from "../HomeGallery/HomeGallery";
import HomeProducts from "../HomeProducts/HomeProducts";

export default function Home() {
  return (
    <>
      <section className="home-intro"></section>
      <HomeDescription />
      <HomeGallery />
      <HomeProducts />
    </>
  );
}
