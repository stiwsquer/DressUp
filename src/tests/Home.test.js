import Home from "../components/Home/Home";
import HomeDescription from "../components/HomeDescription/HomeDescription";
import HomeGallery from "../components/HomeGallery/HomeGallery";
import HomeProducts from "../components/HomeProducts/HomeProducts";
import { BrowserRouter as Router } from "react-router-dom";
import { mount } from "enzyme";

it("renders Home component without crashing", () => {
  shallow(<Home />);
});

it("contains section element", () => {
  const wrapper = shallow(<Home />);
  expect(wrapper.contains(<section className="home-intro"></section>)).toBe(
    true
  );
});

it("contains HomoDescription component", () => {
  const wrapper = shallow(<Home />);
  expect(wrapper.contains(<HomeDescription />)).toBe(true);
});

it("contains HomeGallery component", () => {
  const wrapper = shallow(<Home />);
  expect(wrapper.contains(<HomeGallery />)).toBe(true);
});

it("contains HomeProducts component", () => {
  const wrapper = shallow(<Home />);
  expect(wrapper.contains(<HomeProducts />)).toBe(true);
});

it("Home snapshot - shallow", () => {
  const wrapper = shallow(<Home />);
  expect(wrapper).toMatchSnapshot();
});

it("Home snapshot - render", () => {
  const wrapper = render(
    <Router>
      <Home />
    </Router>
  );
  expect(wrapper).toMatchSnapshot();
});
