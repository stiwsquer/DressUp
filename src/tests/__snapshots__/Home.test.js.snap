// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home snapshot - render 1`] = `
Array [
  <section
    class="home-intro"
  />,
  <section
    class="home-description"
  >
    <h2>
      Trendy Clothing & Accessories at Dress Up - An Online Dress Boutique
    </h2>
    <p>
      DRESS UP IS A WOMEN'S CLOTHING BOUTIQUE CARRYING A VARIETY OF FITS, STYLES, AND PATTERNS FOR WOMEN OF ALL AGES, SHAPES, AND SIZES. WE OFFER A DIVERSE COLLECTION OF NEW PRODUCTS EVERY SINGLE WEEK, INCLUDING YOUR FAVORITE CLOTHING ITEMS: CUTE ROMPERS, MAXI DRESSES, KNIT SWEATERS, OVERSIZED TOPS, AND LIGHTWEIGHT CARDIGANS, AS WELL AS TRENDY SHOES, ACCESSORIES, JEWELRY, ACTIVEWEAR, LOUNGEWEAR, AND SO MUCH MORE. DRESS UP'S TRENDY, YET WEARABLE CLOTHES WILL HAVE YOU FEELING LIKE YOUR BEST SELF, NO MATTER WHERE YOU'RE HEADED. YOU CAN SHOP WITH US ONLINE OR AT ANY OF OUR RETAIL LOCATIONS ACROSS THE SOUTHEAST, INCLUDING STORES IN GEORGIA, ALABAMA, SOUTH CAROLINA, AND TENNESSEE. GET FREE SHIPPING ON ALL ORDERS OVER $49, AND FEEL FREE TO REACH OUT TO US AT 678.989.2006 WITH ANY QUESTIONS. DON'T MISS OUT - COME SEE WHY DRESS UP IS ONE OF THE FASTEST GROWING ONLINE DRESS BOUTIQUES IN THE UNITED STATES!
    </p>
    <a
      href="/"
    >
      <button>
        SHOP OUR NEWEST ARRIVALS
      </button>
    </a>
  </section>,
  <section
    class="home-gallery"
  >
    <div
      class="gallery-new-shoes"
    >
      <h2>
        NEW SHOES
      </h2>
      <p>
        We just dropped new fall shoe styles, and they are all ON SALE! Shop new combat boots, booties, mules, sneakers & more!
      </p>
      <a
        href="/"
      >
        <button>
          SHOP SHOES
        </button>
      </a>
    </div>
    <div
      class="gallery-accessories"
    >
      <h2>
        Accessories
      </h2>
      <p>
        Take your fall outfit to the next level with new accessories! The perfect trendy handbag, dainty gold necklace, coin bracelet or statement earrings will upgrade any look.
      </p>
      <a
        href="/"
      >
        <button>
          SHOP ACCESSORIES
        </button>
      </a>
    </div>
    <img
      class="gallery1"
      src="./img/home/gallery/gallery-1.2.jpg"
    />
    <img
      class="gallery2"
      src="./img/home/gallery/gallery-2.1.jpg"
    />
    <img
      class="gallery3"
      src="./img/home/gallery/gallery-3.1.jpg"
    />
  </section>,
  <section
    class="home-products"
  >
    <div
      class="products-title"
    >
      <span
        class="horizontal-line"
      />
      <h2>
        NEW PRODUCTS
      </h2>
      <span
        class="horizontal-line"
      />
    </div>
    <div
      class="cards"
    >
      <div
        class="sc-jSgvzq jJOCMs rec rec-carousel-wrapper "
      >
        <div
          class="sc-eCstlR bwgQCB rec rec-carousel"
          style="height:0"
        >
          <button
            class="sc-bdfBQB eRmJbc rec rec-arrow rec rec-arrow-left"
            disabled=""
            tabindex="0"
            type="button"
          >
            ‚ùÆ
          </button>
          <div
            class="sc-dlfnuX bVibPY rec rec-slider-container"
          >
            <div
              class="sc-hKgJUU hUugPI rec rec-slider"
              style="transition:all 500ms ease;left:0px;right:auto;top:auto"
            >
              <div
                class="rec rec-swipable"
                style="display:flex;flex-direction:row"
              >
                <div
                  class="rec rec-carousel-item rec-carousel-item-0 rec-carousel-item-visible"
                >
                  <div
                    class="sc-gsTEea eQpvzW rec rec-item-wrapper"
                    id="0"
                    style="width:0px;padding:0px 0px 0px 0px"
                  >
                    <div
                      class="card"
                    >
                      <figure
                        class="card-figure"
                      >
                        <a
                          class="card-image-link"
                          href="/"
                        >
                          <img
                            alt="Image of women in a dress"
                            class=" card-image card-image-normal"
                            src="./img/products/1.JPG"
                          />
                          <img
                            alt="Image of women in a dress"
                            class="card-image  card-image-alt"
                            src="./img/products/1.2.webp"
                          />
                        </a>
                        <figcaption
                          class="fig-caption"
                        >
                          <a
                            href="/"
                          />
                        </figcaption>
                      </figure>
                      <div
                        class="card-body"
                      >
                        <a
                          href="/"
                        >
                          <h4
                            class="card-title"
                          >
                            Fiercely Fabulous Leopard Ruched Mini Dress
                          </h4>
                        </a>
                        <p
                          class="card-text"
                        >
                          $44.00
                        </p>
                      </div>
                      <div
                        class="card-swatches"
                      />
                    </div>
                  </div>
                </div>
                <div
                  class="rec rec-carousel-item rec-carousel-item-1 rec-carousel-item-hidden rec-carousel-item-next"
                >
                  <div
                    class="sc-gsTEea eQpvzW rec rec-item-wrapper"
                    id="1"
                    style="width:0px;padding:0px 0px 0px 0px"
                  >
                    <div
                      class="card"
                    >
                      <figure
                        class="card-figure"
                      >
                        <a
                          class="card-image-link"
                          href="/"
                        >
                          <img
                            alt="Image of women in a dress"
                            class=" card-image card-image-normal"
                            src="./img/products/2.webp"
                          />
                          <img
                            alt="Image of women in a dress"
                            class="card-image  card-image-alt"
                            src="./img/products/2.1.webp"
                          />
                        </a>
                        <figcaption
                          class="fig-caption"
                        >
                          <a
                            href="/"
                          />
                        </figcaption>
                      </figure>
                      <div
                        class="card-body"
                      >
                        <a
                          href="/"
                        >
                          <h4
                            class="card-title"
                          >
                            Tame My Heart Leopard Tie-Front Dress
                          </h4>
                        </a>
                        <p
                          class="card-text"
                        >
                          $40.00
                        </p>
                      </div>
                      <div
                        class="card-swatches"
                      />
                    </div>
                  </div>
                </div>
                <div
                  class="rec rec-carousel-item rec-carousel-item-2 rec-carousel-item-hidden"
                >
                  <div
                    class="sc-gsTEea eQpvzW rec rec-item-wrapper"
                    id="2"
                    style="width:0px;padding:0px 0px 0px 0px"
                  >
                    <div
                      class="card"
                    >
                      <figure
                        class="card-figure"
                      >
                        <a
                          class="card-image-link"
                          href="/"
                        >
                          <img
                            alt="Image of women in a dress"
                            class=" card-image card-image-normal"
                            src="./img/products/3.webp"
                          />
                          <img
                            alt="Image of women in a dress"
                            class="card-image  card-image-alt"
                            src="./img/products/3.1.webp"
                          />
                        </a>
                        <figcaption
                          class="fig-caption"
                        >
                          <a
                            href="/"
                          />
                        </figcaption>
                      </figure>
                      <div
                        class="card-body"
                      >
                        <a
                          href="/"
                        >
                          <h4
                            class="card-title"
                          >
                            Keepin' It Chic Tiered Dress 
                          </h4>
                        </a>
                        <p
                          class="card-text"
                        >
                          $33.00
                        </p>
                      </div>
                      <div
                        class="card-swatches"
                      />
                    </div>
                  </div>
                </div>
                <div
                  class="rec rec-carousel-item rec-carousel-item-3 rec-carousel-item-hidden"
                >
                  <div
                    class="sc-gsTEea eQpvzW rec rec-item-wrapper"
                    id="3"
                    style="width:0px;padding:0px 0px 0px 0px"
                  >
                    <div
                      class="card"
                    >
                      <figure
                        class="card-figure"
                      >
                        <a
                          class="card-image-link"
                          href="/"
                        >
                          <img
                            alt="Image of women in a dress"
                            class=" card-image card-image-normal"
                            src="./img/products/4.webp"
                          />
                          <img
                            alt="Image of women in a dress"
                            class="card-image  card-image-alt"
                            src="./img/products/4.1.JPG"
                          />
                        </a>
                        <figcaption
                          class="fig-caption"
                        >
                          <a
                            href="/"
                          />
                        </figcaption>
                      </figure>
                      <div
                        class="card-body"
                      >
                        <a
                          href="/"
                        >
                          <h4
                            class="card-title"
                          >
                            Walk The Walk Mini Skirt
                          </h4>
                        </a>
                        <p
                          class="card-text"
                        >
                          $39.00
                        </p>
                      </div>
                      <div
                        class="card-swatches"
                      />
                    </div>
                  </div>
                </div>
                <div
                  class="rec rec-carousel-item rec-carousel-item-4 rec-carousel-item-hidden"
                >
                  <div
                    class="sc-gsTEea eQpvzW rec rec-item-wrapper"
                    id="4"
                    style="width:0px;padding:0px 0px 0px 0px"
                  >
                    <div
                      class="card"
                    >
                      <figure
                        class="card-figure"
                      >
                        <a
                          class="card-image-link"
                          href="/"
                        >
                          <img
                            alt="Image of women in a dress"
                            class=" card-image card-image-normal"
                            src="./img/products/5.JPG"
                          />
                          <img
                            alt="Image of women in a dress"
                            class="card-image  card-image-alt"
                            src="./img/products/5.1.JPG"
                          />
                        </a>
                        <figcaption
                          class="fig-caption"
                        >
                          <a
                            href="/"
                          />
                        </figcaption>
                      </figure>
                      <div
                        class="card-body"
                      >
                        <a
                          href="/"
                        >
                          <h4
                            class="card-title"
                          >
                            Just The One Floral Dress - Mustard
                          </h4>
                        </a>
                        <p
                          class="card-text"
                        >
                          $44.00
                        </p>
                      </div>
                      <div
                        class="card-swatches"
                      />
                    </div>
                  </div>
                </div>
                <div
                  class="rec rec-carousel-item rec-carousel-item-5 rec-carousel-item-hidden"
                >
                  <div
                    class="sc-gsTEea eQpvzW rec rec-item-wrapper"
                    id="5"
                    style="width:0px;padding:0px 0px 0px 0px"
                  >
                    <div
                      class="card"
                    >
                      <figure
                        class="card-figure"
                      >
                        <a
                          class="card-image-link"
                          href="/"
                        >
                          <img
                            alt="Image of women in a dress"
                            class=" card-image card-image-normal"
                            src="./img/products/6.webp"
                          />
                          <img
                            alt="Image of women in a dress"
                            class="card-image  card-image-alt"
                            src="./img/products/6.1.webp"
                          />
                        </a>
                        <figcaption
                          class="fig-caption"
                        >
                          <a
                            href="/"
                          />
                        </figcaption>
                      </figure>
                      <div
                        class="card-body"
                      >
                        <a
                          href="/"
                        >
                          <h4
                            class="card-title"
                          >
                            Iconic Style Tie-Front Blouse
                          </h4>
                        </a>
                        <p
                          class="card-text"
                        >
                          $35.00
                        </p>
                      </div>
                      <div
                        class="card-swatches"
                      />
                    </div>
                  </div>
                </div>
                <div
                  class="rec rec-carousel-item rec-carousel-item-6 rec-carousel-item-hidden"
                >
                  <div
                    class="sc-gsTEea eQpvzW rec rec-item-wrapper"
                    id="6"
                    style="width:0px;padding:0px 0px 0px 0px"
                  >
                    <div
                      class="card"
                    >
                      <figure
                        class="card-figure"
                      >
                        <a
                          class="card-image-link"
                          href="/"
                        >
                          <img
                            alt="Image of women in a dress"
                            class=" card-image card-image-normal"
                            src="./img/products/7.JPG"
                          />
                          <img
                            alt="Image of women in a dress"
                            class="card-image  card-image-alt"
                            src="./img/products/7.1.JPG"
                          />
                        </a>
                        <figcaption
                          class="fig-caption"
                        >
                          <a
                            href="/"
                          />
                        </figcaption>
                      </figure>
                      <div
                        class="card-body"
                      >
                        <a
                          href="/"
                        >
                          <h4
                            class="card-title"
                          >
                            Wild Spirit Leopard Babydoll Dress
                          </h4>
                        </a>
                        <p
                          class="card-text"
                        >
                          $41.00
                        </p>
                      </div>
                      <div
                        class="card-swatches"
                      />
                    </div>
                  </div>
                </div>
                <div
                  class="rec rec-carousel-item rec-carousel-item-7 rec-carousel-item-hidden"
                >
                  <div
                    class="sc-gsTEea eQpvzW rec rec-item-wrapper"
                    id="7"
                    style="width:0px;padding:0px 0px 0px 0px"
                  >
                    <div
                      class="card"
                    >
                      <figure
                        class="card-figure"
                      >
                        <a
                          class="card-image-link"
                          href="/"
                        >
                          <img
                            alt="Image of women in a dress"
                            class=" card-image card-image-normal"
                            src="./img/products/8.webp"
                          />
                          <img
                            alt="Image of women in a dress"
                            class="card-image  card-image-alt"
                            src="./img/products/8.1.webp"
                          />
                        </a>
                        <figcaption
                          class="fig-caption"
                        >
                          <a
                            href="/"
                          />
                        </figcaption>
                      </figure>
                      <div
                        class="card-body"
                      >
                        <a
                          href="/"
                        >
                          <h4
                            class="card-title"
                          >
                            Wild Impression Leopard Blouse
                          </h4>
                        </a>
                        <p
                          class="card-text"
                        >
                          $33.00
                        </p>
                      </div>
                      <div
                        class="card-swatches"
                      />
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <button
            class="sc-bdfBQB khvUfi rec rec-arrow rec rec-arrow-right"
            tabindex="0"
            type="button"
          >
            ‚ùØ
          </button>
        </div>
        <div
          class="sc-iBPTik iHAfmq rec rec-pagination"
        >
          <button
            class="sc-gKseQn jJKuoL rec rec-dot rec rec-dot_active"
            tabindex="-1"
            type="button"
          />
          <button
            class="sc-gKseQn gQIqMI rec rec-dot "
            tabindex="0"
            type="button"
          />
          <button
            class="sc-gKseQn gQIqMI rec rec-dot "
            tabindex="0"
            type="button"
          />
          <button
            class="sc-gKseQn gQIqMI rec rec-dot "
            tabindex="0"
            type="button"
          />
          <button
            class="sc-gKseQn gQIqMI rec rec-dot "
            tabindex="0"
            type="button"
          />
          <button
            class="sc-gKseQn gQIqMI rec rec-dot "
            tabindex="0"
            type="button"
          />
          <button
            class="sc-gKseQn gQIqMI rec rec-dot "
            tabindex="0"
            type="button"
          />
          <button
            class="sc-gKseQn gQIqMI rec rec-dot "
            tabindex="0"
            type="button"
          />
        </div>
      </div>
    </div>
  </section>,
]
`;

exports[`Home snapshot - shallow 1`] = `
<Fragment>
  <section
    className="home-intro"
  />
  <HomeDescription />
  <HomeGallery />
  <HomeProducts />
</Fragment>
`;
